src/test/scala/SparkIntegrationTest.scala

```diff
@@ -180,15 +180,14 @@ class SparkIntegrationTest
     WeaviateDocker.deleteClass()
   }
 
-  test("Article with strings and date") {
```

Comment:
could you add another test change this test to Article with string sequences and date? We don't want to lose test coverage for date datatype.

Reply:
donezo